client/index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>SOS Distress</title>
-    <link rel="stylesheet" type="text/css" href="../styles/style.css" />
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script type="text/javascript" src="https://maps.googleapis.com/maps/api/js?libraries=places"></script>
+    <link rel="stylesheet" type="text/css" href="../styles/style.css" />
+    <link rel="stylesheet" href="./lib/bower_components/intl-tel-input/build/css/intlTelInput.css">
```

Comment:
I see there are also changes to style.css. What's the relation between those changes and what's in intlTelInput.css?